### ISSUE TITLE
TypeError: "Cannot read property 'PYTHON' of undefined" when using --build

### ISSUE THREAD

**Issue by don** [nexe/nexe#486]
This is a
 - [x] Bug Report
 - [ ] Feature Request
 - [ ] Other

<br/>

When I try  packaging with nexe using the `--build` flag, I get `TypeError: Cannot read property 'PYTHON' of undefined`. I see this on Linux ARM and macOS.

Steps to duplicate

	$ mkdir hello
	$ cd hello
	$ npm init -y
	$ echo console.log\(\'hello\'\) > index.js
	$ npx nexe index.js --build

TypeError: Cannot read property 'PYTHON' of undefined

I can work around this by manually building the downloaded source

    $ cd ~/.nexe/8.10.0/
    $ ./configure && make
    $ cd ~/hello
    $ npx nexe index.js --build

Do I need to set some additional configuration or is this a bug?
<br/><br/>
Please also provide:

- Platform(OS/Version): macOS 10.13.4 and Linux rpi ARM
- Host Node Version:  8.10.0
- Target Node Version: 8.10.0
- Nexe version:  nexe@2.0.0-rc.28 
- Python Version: 2.7.10 (macOS) 2.7.13 (Linux ARM)

**Comment by ramkumarc** [comment]
I had the same issue. Adding `-p /usr/bin/python` worked in my macOS 10.13.4. The path i specified is from `which python` command. Hope this helps.

**Comment by beenotung** [comment]
The suggestion from ramkumarc doesn't works on (Arch) Linux :(
```
$ nexe -p /usr/bin/python --build index.js 
ℹ nexe 2.0.0-rc.28
RUN
✔ Source already downloaded
⠋ Compiling result
Error: /usr/bin/python ./configure --dest-cpu=x64 exited with code: 1
    at ChildProcess.<anonymous> (/home/beenotung/.config/yarn/global/node_modules/nexe/lib/compiler.js:206:28)
    at Object.onceWrapper (events.js:273:13)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:961:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:5)
✔ Configuring node build: --dest-cpu=x64
✔ Finished in 0.521s
Error: /usr/bin/python ./configure --dest-cpu=x64 exited with code: 1
    at ChildProcess.<anonymous> (/home/beenotung/.config/yarn/global/node_modules/nexe/lib/compiler.js:206:28)
    at Object.onceWrapper (events.js:273:13)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:961:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:5)
```

**Comment by don** [comment]
@beenotung if manually compile the source in ~/.nexe/8.10.0/, nexe will find and used the compiled code next time you package an app

**Comment by calebboyd** [comment]
closed via 089aa02a8208907db0a4145715896a72383197e1

**Comment by beenotung** [comment]
I tried to git clone the master branch,
then built with yarn and npm.
Both case failed to build, below is the console output:
```
/home/beenotung/tmp/p/node-bin/node_modules/nexe/tasks/post-build.ts:1
(function (exports, require, module, __filename, __dirname) { import { writeFileSync, readFileSync } from 'fs'
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Module._extensions..js (module.js:663:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/home/beenotung/tmp/p/node-bin/node_modules/nexe/node_modules/ts-node/src/index.ts:384:14)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at Object.<anonymous> (/home/beenotung/tmp/p/node-bin/node_modules/nexe/node_modules/ts-node/src/_bin.ts:182:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nexe@2.0.0-rc.29 postbuild: `ts-node tasks/post-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nexe@2.0.0-rc.29 postbuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
